README.md

```diff
@@ -17,9 +17,9 @@ NetKet is an open-source project delivering cutting-edge methods for the study
 of many-body quantum systems with artificial neural networks and machine learning techniques.
 It is a Python library built on [JAX](https://github.com/google/jax).
 
-- **Homepage:** <https://netket.org>
+- **Homepage:** <https://www.netket.org>
```

Comment:
```suggestion
- **Homepage:** <netket.org>
```

Reply:
The `https://` is needed, otherwise the link doesn't work.

Using `https://netket.org` is fine by me, but then we need to make it work again as right now it's unavailable.